database/src/main/postgres/runs/additional_data.ddl

```diff
@@ -21,12 +21,10 @@ CREATE TABLE runs.additional_data
     ad_value                TEXT,
     created_by              TEXT NOT NULL,
     created_at              TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
-    updated_by              TEXT NOT NULL,
```

Comment:
hmmm. I knew that you will say this 😄 it's not that I didn't consider this - I did. 

I consider the upsert to actually more like a rewrite (delete and insert), meaning that basically the authorship will be rewritten (created_by but also created_at). This is what I had the biggest fight in my head internally with - if it would be just a regular update, then probably only AD value alone would change, not the information about original creation. But to me, this is good to be overwritten - I want to know who created the key->value pair, not just key.

Happy to discuss over a call